fbw-a380x/src/systems/instruments/src/MFD/pages/common/DataEntryFormats.tsx

```diff
@@ -435,7 +435,16 @@ export class TemperatureFormat implements DataEntryFormat<number> {
       return null;
     }
 
-    const nbr = Number(input);
+    let nbr = Number(input);
+    
+    if (nbr > 0 && input.substring(0, 1) !== '+') {
```

Comment:
This doesn't seem to make much sense. This will make everything negative unless you explicitly prefix with +.